packages/os/pluto.service

```diff
@@ -17,4 +18,4 @@ RemainAfterExit=true
 StandardError=journal+console
 
 [Install]
-RequiredBy=preconfigured.target
+RequiredBy=settings-applier.service
```

Comment:
I'd prefer to comment here that pluto requires the settings-applier service to commit its changes added to an open transaction.

This may be a clear responsibility implied by its position in the service tree that I've forgotten, though. If it's an obvious detail we can omit.

Reply:
> I'd prefer to comment here that pluto requires the settings-applier service to commit its changes added to an open transaction.

Added some comments. Let me know if it's still not clear; this is an under-documented area of the system.